packages/edit-site/src/components/revisions/index.js

```diff
@@ -56,7 +56,9 @@ function Revisions( { userConfig, blocks } ) {
 		[ originalSettings ]
 	);
 
-	const [ globalStyles ] = useGlobalStylesOutputWithConfig( mergedConfig );
+	const [ globalStyles ] = useGlobalStylesOutputWithConfig( mergedConfig, {
```

Comment:
I'm guessing the Style Book should also render these variations styles:

https://github.com/WordPress/gutenberg/blob/7f7128eeec0f19ffa86621cb993bd139ef163946/packages/edit-site/src/components/style-book/index.js#L230-L230

I tried it quickly, but I'm not yet sure it's enough - the preview doesn't render the style variation CSS, or update it when editing the global block styles.

Could be missing something. I'll look next week. Might be a different PR.